x/tokenomics/keeper/settle_pending_claims.go

```diff
@@ -7,7 +7,6 @@ import (
 	cosmoslog "cosmossdk.io/log"
```

Comment:
1. No impact on unit tests?
2. If not - should it have an impact on unit tests?

Reply:
It does not and should not have an impact on unit tests, it's a mere replacement of `AllClaims` querier.

The only core change to `SettlePendingClaims` is about how expiring claims are iterated over:
```diff
-	for _, claim := range expiringClaims {
+	for ; expiringClaimsIterator.Valid(); expiringClaimsIterator.Next() {
```